README.md

```diff
@@ -82,3 +84,17 @@ We appreciate feedback and contribution to this repo!
 This project is licensed under the Apache-2.0 license. See the
 [LICENSE](https://github.com/authts/oidc-client-ts/blob/main/LICENSE) file for
 more info.
+
+## Security concerns on Resource Owner Password Credentials flow
```

Comment:
Please move that **great** but huge text into `docs/index.md` or create a new file in `docs/` and link directly from

`- [Resource Owner Password Credentials Grant](https://www.rfc-editor.org/rfc/rfc6749#section-1.3.3); however, read the security concerns below before using this flow`
->
`- [Resource Owner Password Credentials Grant](https://www.rfc-editor.org/rfc/rfc6749#section-1.3.3); however, read the [security concerns](link...) before using this flow`